src/qibosoq/client.py

```diff
@@ -32,22 +34,35 @@ def connect(server_commands: dict, host: str, port: int) -> Tuple[list, list]:
                 break
             received.extend(tmp)
         results = json.loads(received.decode("utf-8"))
-        if isinstance(results, str) and "Error" in results:
+        if isinstance(results, str):
             raise QibosoqError(results)
         return results["i"], results["q"]
 
 
+def serialize_sweeper(sweeper: Sweeper) -> dict:
```

Comment:
Since this is a function of a single argument, i.e. the Qibosoq class `Sweeper`, this would work perfectly as a `Sweeper` property, e.g. `.dict`/`.serialized`/`.raw`.

The main advantage is that you avoid the need for a further import.